packages/nextjs/test/integration/package.json

```diff
@@ -33,6 +33,7 @@
     "@sentry/replay": "file:../../../replay",
     "@sentry/tracing": "file:../../../tracing",
     "@sentry-internal/tracing": "file:../../../tracing-internal",
+    "@sentry-internal/feedback": "file:../../../feedback",
```

Comment:
Question: Why doesn't this change make any updates on `yarn.lock`?

Reply:
this `package.json` is not part of the workspace, it's nested inside the tests.

I think we should avoid this and move the nextjs tests out, but it's something we can refactor later on.

Reply:
yup, same for remix